In [12]:
import numpy as np
from math import factorial

#### Задание 1.
##### Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

$$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$$
$2Ф(t_{\lambda}) = 0,95$  
Из таблицы Лапласа находим $t_{\lambda} = 1,96$  
$\sigma = 16$  
$n = 256$

In [23]:
delta = (1.96 * 16) / np.sqrt(256)
delta

1.96

In [25]:
80 - delta, 80 + delta

(78.04, 81.96)

Доверительный интервал будет от 78.04 до 81.96

#### Задание 2.
##### В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

Воспользуемся формулой:
$$P(M - \delta < \alpha < M + \delta) = 1 - \lambda$$
$n = 10$  
$\lambda = 1 - 0,95 = 0,05$  
$M = \frac{6,9 + 6,1 + 6,2 + 6,8 + 7,5 + 6,3 + 6,4 + 6,9 + 6,7 + 6,1}{10}$  
$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$  
Из таблиц распределения Стьюдента при "n-1" и $\lambda = 0,05$ :  
$t_{\lambda} = 2,262$

In [35]:
test_1 = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
sigma2 = np.std(test_1, ddof=1)
delta_2 = 2.262 * sigma2 / np.sqrt(10)
delta_2

0.32246174408757394

In [37]:
m_2 = (6.9 + 6.1 + 6.2 + 6.8 + 7.5 + 6.3 + 6.4 + 6.9 + 6.7 + 6.1) / 10
m_2

6.589999999999999

In [39]:
m_2 - delta_2, m_2 + delta_2

(6.267538255912425, 6.912461744087573)

Истинное значение случайной величины лежит в доверительном интервале (6.267538255912425; 6.912461744087573) с доверительной вероятностью 0,95.

#### Задание 3.
##### Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

##### Гипотеза "H0": Математическое ожидание MX = 17

##### Гипотеза "H1":  Математическое ожидание MX > 17

Уровень значимости:  
$\alpha = 0,05$  
Дисперсия:  
$D(X) = \sigma ^ 2 = 4$  
Выборка:  
$n = 100$  
Средний диаметр шариков(измеренный):  
$\overline{x} = 17,5$  
Функция вычисления наблюдаемого значения критерия:  
$U_{набл} = \frac{(\overline{x} - \mu)}{\sigma}\sqrt{n}$

In [31]:
n = 100
xsr = 17.5
mx = 17
sigma3 = np.sqrt(4)
alfa = 0.05
un = (xsr - mx) / sigma3 * np.sqrt(n)
un

2.5

Из таблицы Лапласа найдем критическую точку для односторонней критической области:

In [21]:
f = (1 - 2*alfa) / 2
f

0.45

$U_{кр} \approx 1.65$

Так как $U_{набл}$ = 2,5 > 1.65 = $U_{кр}$, то гипотезу H1 можно принять и считать, что средний диаметр шариков равен 17,5 мм.

#### Задание 4.
##### Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?